lib/web/command_test.go

```diff
@@ -47,9 +51,17 @@ import (
 	"github.com/gravitational/teleport/lib/utils"
 )
 
+const testCommand = "echo txlxport | sed 's/x/e/g'"
```

Comment:
This command is getting popular :laughing: 
```bash
$ grep -HR "echo txlxport | sed 's/x/e/g'" . | wc -l
4
```

Reply:
I changed everything to use this new constant instead fo the string. However this makes the flaky test detector run on Terminal tests (and they seem racy). I might have to undo this change to merge the branch.

Reply:
+1 for reverting the change. Fixing the tests shouldn't be part of this PR.